src/components/DataTableSkeleton/DataTableSkeleton.js

```diff
@@ -18,7 +18,7 @@ const DataTableSkeleton = ({
 
   const rows = [];
   const rowRepeat = rowCount - 1;
-  const columnsArray = Array.from(Array(columnCount).keys()); // [0,1,2...columnCount-1]
+  const columnsArray = Array.from(Array(columnCount).map((i, idx) => idx)); // [0,1,2...columnCount-1]
```

Comment:
Could we update this to:

```suggestion
  const columnsArray = Array.from({ length: 5 }, (_, index) => index)
```

Reply:
I changed it to use the suggested syntax, but with the prop's value instead of a hardcoded 5. Looks good to you now?